GSA/digitalgov.gov
# Create guides landing page

## Summary
Adds the landing page for the guides section.

<details><summary>Screenshot</summary>
<p>

![Guides _ Digital_gov · 1 01pm · 06-28](https://github.com/GSA/digitalgov.gov/assets/104778659/db63116b-99db-4ba2-9c96-b553dede3634)


</p>
</details> 

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-create-guides-page/guides/)

### Solution

- adds a landing page for all top level guides (does not display HCD sub-guides)
- uses same `usa-card` component code from the HCD guides for consistency
- adds `dg-guides` and `dg-guides-card` component classes for displaying each guide
- adds front matter field `guide_weight` to each guide to display ASC order (lowest to highest) only on the guides landing page


### How To Test

1. Visit [/guides](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-create-guides-page/guides/)
2. Should see guides listed in card list
3. Clicking on each button will visit the guide

---

### Content 

@bonnieAcameron Here are some outstanding content related items:

- [x] update the `guide_weight` to change order, 1 will display first
- [x] update title to remove "Guide to..."
- [x] update summary for each guide?

---

### Dev Checklist
